src/components/settings/FeeLimitDialog.tsx

```diff
@@ -212,99 +173,104 @@ export const FeeLimitDialog = ({ open, onOpenChange, walletFileName }: FeeLimitD
           </DialogDescription>
         </DialogHeader>
 
-        <div className="min-h-0 flex-1 space-y-4">
-          <div className="flex items-center justify-between">
-            <div className="flex items-center gap-3">
-              <Switch checked={enableFormValidation} onCheckedChange={setEnableFormValidation} />
-              <div className="flex items-center gap-2">
-                <span className="text-sm font-medium">Enable form validation</span>
-                <DevBadge />
+        <div className="flex-1 space-y-4">
+          {isDeveloperMode && (
+            <>
+              <div className="flex items-center gap-3">
+                <Switch checked={enableFormValidation} onCheckedChange={setEnableFormValidation} />
+                <div className="flex items-center gap-2">
+                  <span className="text-sm font-medium">Enable form validation</span>
+                  <DevBadge />
+                </div>
               </div>
-            </div>
-          </div>
-          <p className="text-muted-foreground text-sm">
-            Ability to reset fee values to test what the UI looks like, when a user does not have these values
-            configured.
-          </p>
+              <p className="text-muted-foreground text-sm">
+                Ability to reset fee values to test what the UI looks like, when a user does not have these values
+                configured.
+              </p>
+            </>
+          )}
 
           <div className="space-y-2">
-            <button
-              onClick={() => setCollaboratorFeesExpanded(!collaboratorFeesExpanded)}
-              className={`hover:bg-muted/50 flex w-full cursor-pointer items-center justify-between border-b px-4 py-2 text-left ${
-                collaboratorFormRef.current && !collaboratorFormRef.current.getFormData() ? 'border-red-300' : ''
-              }`}
+            <Accordion
+              defaultValue="collaborator-fees"
+              type="single"
```

Comment:
The accordion isn't collapsing for both collaborator and mining fees form
the issue is Radix UI's Accordion component with type="single" doesn't allow collapsing by default. we need to explicitly set collapsible={true} (or just collapsible)

```suggestion
              type="single"
              collapsible
```

Reply:
Same as above.